docker/oauthlib/Dockerfile

```
@@ -0,0 +1,11 @@
+
+FROM demisto/python3:3.7.4.1150
```

```suggestion
FROM demisto/crypto:1.0.0.1024
```

If you use our crypto docker as a base you don't need to install libffi and openssl and the dockers shipped are smaller and optimised.